File: javascript/glicose.js
const glicose = parseFloat(prompt("Digite a medida da glicose"))

if (glicose > 140) {
    console.log("Classificacao: diabetes");
} else if (glicose > 100) {
    console.log("Classificacao: elevado");
} else {
    console.log("Classificacao: normal");
}

File: python/glicose.py
glicose = float(input("Digite a medida da glicose: "))
classificacao = ""

if glicose <= 100:
    classificacao = "normal"
elif glicose <= 140:
    classificacao = "elevado"
else:
    classificacao = "diabetes"

print(f"Classificacao: {classificacao}")
